config/routes.rb

```diff
@@ -4,17 +4,19 @@
   devise_for :buyers
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'pages#home'
-
   get 'artists/:id' => 'artists#show', as: :artistid
-  get '/artists/', to: 'artists#all_artists'
+  get 'artists/', to: 'artists#all_artists'
+  get 'works/' => 'works#index'
+  get 'works/categories' => 'works#get_work_category_enums'
   get 'buyers/:id' => 'buyers#show', as: :buyerid
 
   namespace :api, defaults: { format: :json } do
-    resources :works, :only => [:show, :create, :update, :destroy]
+    resources :works, :only => [:index, :create, :update, :destroy]
```

Comment:
We should restructure the `works/:id/show` method to be part of this line. The structure of the route should instead be `works/show/:id`, and then we can include `show` in this resource mapping the way that it's done for  artists below